### PR TITLE
SDIT-2167 Remove alerts, adjustments, adjudications, migration code

### PR DESCRIPTION
**Pull request by andymarke**
Also removed migration code for:
Sentence adjustments
Adjudications (which is one way sync so removed 100%)

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt [hunk 1]
```diff
@@ -38,23 +28,6 @@ class AlertsNomisApiService(@Qualifier("nomisApiWebClient") private val webClien
...
 
-  suspend fun getAlertsByBookingId(bookingId: Long): BookingAlertsResponse = webClient.get()
-    .uri(
-      "/prisoners/booking-id/{bookingId}/alerts",
-      bookingId,
-    )
-    .retrieve()
-    .awaitBody()
-
-  suspend fun getBookingPreviousTo(offenderNo: String, bookingId: Long): PreviousBookingId = webClient.get()
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt:49]: also not used

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesByPrisonerMigrationIntTest.kt [hunk 2]
```diff
@@ -28,10 +28,10 @@ import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.http.MediaType
 import org.springframework.test.web.reactive.server.returnResult
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.casenotes.CaseNotesApiExtension.Companion.caseNotesApi
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.helper.MigrationResult
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesByPrisonerMigrationIntTest.kt:31]: this test class was be pulled in from sentence adjustments so moved to own file

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationIntTest.kt [hunk 3]
```diff
@@ -86,7 +85,7 @@ class PrisonPersonMigrationIntTest : SqsIntegrationTestBase() {
         webTestClient.post().uri("/migrate/prisonperson")
           .headers(setAuthorisation(roles = listOf()))
           .contentType(MediaType.APPLICATION_JSON)
-          .bodyValue(AlertsMigrationFilter())
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationIntTest.kt:89]: was using this by mistake so just switched to an empty filter

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**